core/arch/instr.h

```diff
@@ -1911,6 +1924,19 @@ DR_API
 dr_instr_label_data_t *
 instr_get_label_data_area(instr_t *instr);
 
+DR_API
+/**
+ * Set a function \p func which is called when the label instruction is freed.
+ * \p instr is typically an instruction related to the label which can be freed
+ * at the same time as the label.
```

Comment:
Again, the 2nd sentence is not correct.

Reply:
Oops, good spot! Misleading for users! Fixed.